main.go

```diff
@@ -116,6 +116,8 @@ func (*requesthandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 
 	path := req.URL.EscapedPath()
 
+	path = strings.Replace(path, path_prefix, "", 1)
```

Comment:
this is needed to avoid `rewrite` rule in nginx to strip directly the `path_prefix` from the uri before the `proxy_pass` directive.

`rewrite` rule has url encode strategy that breaks proxying the request:
compare right one:
```
yes/mime/audio%2Fwebm/otfp/1/gir/yes/clen/329690/lmt/1563164106985925/dur/16.181/mt/1637516647/fvip/2/keepalive/yes/fexp/24001373,24007246/sparams/expire,ei,ip,id,itag,source,requiressl,vprv,ratebypass,mime,otfp,gir,clen,lmt,dur/sig/AOq0QJ8wRQIgBWYGgfj12wxMOYPKd5x1nisf_3OtsttmD4WH21Tf17YCIQDf_UjpCyLBZ8zBBbnB2nJunrVvwceEohR_PZMuo-58Tw%3D%3D/lsparams/mh,mm,mn,ms,mv,mvi,pl/lsig/AG3C_xAwRQIhAMTVmMWXtnd0kA-EVVomXjJuHk9UAigCD61ujSPzYeR7AiA4msazYjob1b0OB337ynirUMIBQ0V5dSH1dHsf-KX0yA%3D%3D/range/0-258
```

wrong one with `rewrite` rule:
```
yes/mime/audio/webm/otfp/1/gir/yes/clen/329690/lmt/1563164106985925/dur/16.181/mt/1637516647/fvip/2/keepalive/yes/fexp/24001373,24007246/sparams/expire,ei,ip,id,itag,source,requiressl,vprv,ratebypass,mime,otfp,gir,clen,lmt,dur/sig/AOq0QJ8wRQIgBWYGgfj12wxMOYPKd5x1nisf_3OtsttmD4WH21Tf17YCIQDf_UjpCyLBZ8zBBbnB2nJunrVvwceEohR_PZMuo-58Tw==/lsparams/mh,mm,mn,ms,mv,mvi,pl/lsig/AG3C_xAwRQIhAMTVmMWXtnd0kA-EVVomXjJuHk9UAigCD61ujSPzYeR7AiA4msazYjob1b0OB337ynirUMIBQ0V5dSH1dHsf-KX0yA==/range/0-258
```